data/phones/README.md

```diff
@@ -36,7 +36,13 @@ We welcome user submissions for `.phones` files from linguists. Note that we use
 
 1.  Make a list of all phones or phonemes, in descending-frequency order, using
     the appropriate file in [`../tsv`](../tsv).
+    The script `list_phones.py` is available to facilitate this step.
+    Running `./list_phones.py ../tsv/<some-TSV-file> > foo.phones`
+    generates `foo.phones` that you can edit by the following steps.
 2.  Remove typos, invalid IPA transcriptions, and non-native segments.
+    The `.phones` file generated by `list_phones.py` shows the frequency of each
+    phone/phoneme and several of its example word-pronunciation pairs,
+    which should help you decide which phones/phonemes to remove.
```

Comment:
Maybe remind them to remove comments and counts.

It occurs to me that maybe to be sure, you could comment out the counts too?